src/CDSAPI.jl

```diff
@@ -52,12 +53,22 @@ function retrieve(name, params::AbstractDict, filename; wait=1.0)
     data = JSON.parse(String(response.body))
     endpoint = Dict(response.headers)["location"]
 
+    laststatus = nothing
     while data["status"] != "successful"
+
         data = HTTP.request("GET", endpoint,
             ["PRIVATE-TOKEN" => creds["key"]]
         )
         data = JSON.parse(String(data.body))
-        @info "CDS request" dataset=name status=data["status"]
+
+        if data["status"] != laststatus
+            print("\e[1K\e[0E") # erase terminal line and reset cursor
```

Comment:
Please make sure you are using portable methods to erase the terminal. The REPL module doesn't provide a function for that?

Reply:
It's portable on most modern terminals, not supported on terminals like windows `cmd.exe`.
This is kind of a gimmick anyway. So it's probably better to just remove it.